tests/precompiles/test_precompiles.py

```diff
@@ -0,0 +1,341 @@
+import pytest
+
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+
+@pytest.mark.parametrize(
+    ["address", "exists"],
+    [
+        ("01", True),
+        ("02", True),
+        ("03", True),
+        ("04", True),
+        ("05", True),
+        ("06", True),
+        ("07", True),
+        ("08", True),
+        ("09", True),
+        ("0A", True),
+        ("0B", False),
```

Comment:
This list would only be valid for Cancun, but we could use this marker [`@pytest.mark.parametrize_by_fork`](https://ethereum.github.io/execution-spec-tests/main/writing_tests/test_markers/#pytestmarkparametrize_by_fork) to dynamically generate this list by fork.

I would also reduce the amount of cases to the first `exists=False` fork.

Reply:
yes. but what if we will have gaps? like new precompiles defined 0x20, 0x21, 0x22 but not at 0x1f, 0x1e